dart/lib/src/sentry_measurement.dart

```diff
@@ -1,34 +1,52 @@
+import 'sentry_measurement_unit.dart';
+
 class SentryMeasurement {
-  SentryMeasurement(this.name, this.value);
+  SentryMeasurement(
+    this.name,
+    this.value, {
+    this.unit,
+  });
 
   /// Amount of frames drawn during a transaction
-  SentryMeasurement.totalFrames(this.value) : name = 'frames_total';
+  SentryMeasurement.totalFrames(this.value)
+      : name = 'frames_total',
+        unit = SentryMeasurementUnit.none;
```

Comment:
`m`: I'm a bit confused that this is `none`. Why not `int`?

Reply:
Its part of the protocol linked in the PR description.
https://github.com/getsentry/relay/blob/0d825c0e259ff6759e5c49f6412178f4768c8516/relay-server/src/metrics_extraction/transactions.rs#L248-L258